<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPARK Secrétariat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Kholleurs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_kholleurs">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_kholleurs"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>463</width>
           <height>496</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Classes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_classes">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_classes"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_publish">
        <property name="text">
         <string>Diffuser</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_File_Create"/>
    <addaction name="action_File_Open"/>
    <addaction name="separator"/>
    <addaction name="action_File_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_File_Quit"/>
   </widget>
   <widget class="QMenu" name="menuBDD">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;BDD</string>
    </property>
    <addaction name="action_DB_Classes"/>
    <addaction name="action_DB_Kholleurs"/>
    <addaction name="action_DB_Links"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_AboutIt"/>
   </widget>
   <widget class="QMenu" name="menuDiffusion">
    <property name="title">
     <string>Diffusion</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionDiffuser"/>
    <addaction name="actionImprimer"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuBDD"/>
   <addaction name="menuDiffusion"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_File_Open">
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_File_Create">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C&amp;réer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_File_Delete">
   <property name="text">
    <string>&amp;Supprimer</string>
   </property>
  </action>
  <action name="action_File_Configuate">
   <property name="text">
    <string>&amp;Configurer</string>
   </property>
  </action>
  <action name="action_File_Quit">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_DB_Students">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Elèves</string>
   </property>
  </action>
  <action name="action_DB_Groups">
   <property name="text">
    <string>&amp;Groupes</string>
   </property>
  </action>
  <action name="action_DB_Classes">
   <property name="text">
    <string>Classes</string>
   </property>
  </action>
  <action name="action_DB_Kholleurs">
   <property name="text">
    <string>Kholleurs</string>
   </property>
  </action>
  <action name="action_Schedule_Timetable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Emploi du temps</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Schedule_Students_Groups">
   <property name="text">
    <string>&amp;Liens élèves-groupes</string>
   </property>
  </action>
  <action name="action_Schedule_Kholles">
   <property name="text">
    <string>&amp;Horaires de kholles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_Schedule_Events">
   <property name="text">
    <string>Evènements &amp;spéciaux</string>
   </property>
  </action>
  <action name="action_Kholles_Interface">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Interface</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Kholles_Generate">
   <property name="text">
    <string>&amp;Générer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Kholles_Historic">
   <property name="text">
    <string>&amp;Historique</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Contacts</string>
   </property>
  </action>
  <action name="action_AboutIt">
   <property name="text">
    <string>A p&amp;ropos de</string>
   </property>
  </action>
  <action name="action_Kholles_LastChanges">
   <property name="text">
    <string>&amp;Outils de dépannage</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Schedule_Swapping_Groups">
   <property name="text">
    <string>Echange de &amp;groupes</string>
   </property>
  </action>
  <action name="action_File_Settings">
   <property name="text">
    <string>P&amp;aramètres</string>
   </property>
  </action>
  <action name="action_Kholles_QR_Code">
   <property name="text">
    <string>QR Code</string>
   </property>
  </action>
  <action name="action_DB_Links">
   <property name="text">
    <string>Liens classes-kholleurs</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionDiffuser">
   <property name="text">
    <string>Diffuser</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
  <action name="actionTaper_ici_2">
   <property name="text">
    <string>Taper ici</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
